torchgeo/samplers/single.py

```diff
@@ -178,20 +203,49 @@ def set_worker_split(self, total_workers: int, worker_num: int) -> None:
         """
         self.chips = np.array_split(self.chips, total_workers)[worker_num]
 
-    def save(self, path: str, driver: str) -> None:
+    def save(self, path: str, driver: str = "Shapefile") -> None:
         """Save the chips as a shapefile or feather file"""
```

Comment:
Not part of your PR, but should actually be changed to 
```suggestion
        """Save the chips as a file supported by any geopandas driver or feather file"""
```